eegio/loaders/bids/basebids.py

```diff
@@ -6,20 +6,18 @@
 
 import os
 import warnings
-from collections import OrderedDict
-from typing import Union
 
 import bids
 import bids.reports
 import mne_bids
+from mne_bids.utils import _write_json
 from mne_bids.utils import print_dir_tree
 
-from eegio.base.utils.bids_helper import write_json
-
 
 class BaseBids(object):
     """
-    An instantiator class of the bids format that any BIDS derivative would use.
+    An instantiator class of the bids format that extends usage of BIDS
```

Comment:
In general, the summary statement for the docstring format needs to be one line and end with a period. You can add a line with more detail after.